README.md

```diff
@@ -23,54 +23,31 @@ iPSL is part of the [iTesla Tool](https://github.com/itesla/ipst), and thus, it
 Therefore, the SmarTS Lab / ALSETLab team decided to create the OpenIPSL fork in order to develop the library in a direction that is more suitable for researchers and teachers/professors, and in a transparent, open source software approach.
 
 ## Documentation:
-Documentation is provided within the library and can be accessed when loading OpenIPSL in any Modelica-compliant tool. Additional documentation can also be found [**here**](http://openipsl.readthedocs.io/).
-
-Tutorials on OpenIPSL have been given at many conferences, and are available under [Release](https://github.com/SmarTS-Lab/OpenIPSL/releases).
-We recommend you start from there.
+Documentation is provided within the library and can be accessed when loading OpenIPSL in any Modelica-compliant tool. Tutorials on OpenIPSL have been given at many conferences, and are available under [Release](https://github.com/OpenIPSL/OpenIPSL/releases). We recommend you start from there.
 
 Our documentation assumes that you have working knowledge of the Modelica language, are familiar with a Modelica-compliant modeling and simulation environment, that you have proeficient knowledge on power system steady state analysis (i.e., the so called "power flow"), and knowledge on power system dynamic modeling (i.e., the called "transient" and "small-signal" stability). As such, the documentation is limited, and aims to provide very consice information for people who fulfill the requirements above. If you do not fulfill these requirements, we recommend that you first get acquainted with both Modelica and Power Systems.
 
-
-## Contributing:
-Contributions to the library are welcome and can be submitted in the form of pull requests to this repository.
-
-Please consult the [contribution guidelines](http://openipsl.readthedocs.io/en/latest/community/contributing.html).
-
-More information about our contributors can be found over [here](http://openipsl.readthedocs.io/en/latest/community/community-home.html).
-
-Don't hesitate to contact us.
-
 ## Citing OpenIPSL in Publications:
 If you use OpenIPSL in your work or research, all we ask you in exchange is that you **cite the reference publications**, according to your use.
-Please consult our [Publication List](http://openipsl.readthedocs.io/en/latest/publications.html), for browsing the reference publications.
+Please consult our publication list, located within the Users' Guide package, in the Publications page, for browsing the reference publications.
```

Comment:
I think this whole section can be replaced by use of [CITATION files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files).

Reply:
I checked their documentation and I believe that if we use CITATION.cff, it will not be able to create a citation for the PAPER itself, but the REPO. You see, for example, that the information is translated into @software{} in BibTeX and I am not sure that it is a good idea.

Reply:
Maybe @lvanfretti can also give his input in this matter.

Reply:
WAIT - I think we can configure it. Sorry. Let me check it first.

Reply:
Okay, I have created a cff file...